config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -1,19 +1,18 @@
+# THIS FILE HAS BEEN AUTOMATICALLY GENERATED
+# Don't manually edit it; instead edit the "cmrel" tool which generated it
+# Generated with: /home/ashley/workspace/cert-manager-release/bin/cmrel generate-tests --mode current
 presubmits:
   cert-manager/cert-manager:
-    # Why do we have only have presubmits on "master" but not on the
-    # to-be-released branch, e.g. "release-1.9"? Because we don't need to be
-    # testing e.g. release-1.9 before we have made the first alpha release, e.g.,
-    # "1.9.0-alpha.0". See Step 13.3 in
-    # https://cert-manager.io/docs/contributing/release-process/
     - agent: kubernetes
       always_run: true
       annotations:
+        description: Runs unit and integration tests and verification scripts
+        testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
```

Comment:
note: description is indented incorrectly in the original. This is another bug which is fixed and a good example of why these tests should be generated rather than edited by hand.

(and maybe also an example of why YAML isn't my favourite thing :angel: )

Reply:
Agree, YAML is human-readable but not human-writable without a schema / linter.